packages/expo-in-app-purchases/src/InAppPurchases.types.ts

```diff
@@ -0,0 +1,55 @@
+export type ValidItemType = 'inapp' | 'subs';
+
+export interface QueryResponse {
+  responseCode: ResponseCode,
+  results?: Array<Purchase | ItemDetails>,
+  errorCode?: ErrorCode,
+}
+
+export enum ResponseCode {
+  OK = 0,
+  USER_CANCELED = 1,
+  ERROR = 2,
+  DEFERRED = 3,
+}
+
+export enum ErrorCode {
+  UNKNOWN = 0,
```

Comment:
Thank you for making these enums consistent on the TS side and doing native conversions! 😍 
Can we also make them UpperCamelCase style? That's what TypeScript prefers and uses in its documentation. What do you think @ide?

Reply:
We use FULL_UPPERCASE in the Expo SDK and are intentionally using this convention in the SDK audit. The part from the TS style guide we keep is using CamelCase for the enum types and using singular for enums that are not meant to be combined together and plural for the ones that are.

Reply:
Does FULL_UPPERCASE really make sense? They look terrible, they are less comfortable in writing and finally, I haven't seen any TypeScript convention to write them in this way - all style guides recommends UpperCamelCase/PascalCase. We still use FULL_UPPERCASE in most places, but I assumed it was due to naming conventions we used before we migrated to TypeScript.

Two examples other than TS docs page:
- https://basarat.gitbooks.io/typescript/docs/styleguide/styleguide.html#enum
- https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names

Reply:
Isn't FULL_UPPERCASE a convention that is used for constants, in programming in general?